yw_website/website/templates/pages/detailed_workflow.html

```diff
@@ -1,88 +1,5 @@
-{% load staticfiles %}
```

Comment:
i think the inheritance is a little wierd here. how does this reduce redundancy?